docs/root/operations/admin.rst

```diff
@@ -146,6 +146,12 @@ modify different aspects of the server:
   down or doing a full restart. Invoking this command will universally fail health check requests
   regardless of how the filter is configured (pass through, etc.).
 
+  This also makes :ref:`router filter <config_http_filters_router>` to respond the
```

Comment:
It would be more clear if we reword it like this (with appropriate ref links) - if this what you want to highlight here
 "The health checking filter will automatically set the  x-envoy-immediate-health-check-fail header for subsequent health checks. The router filter will respond to the x-envoy-immediate-health-check-fail header by immediately marking the host as being failed for active health check. Note that this only occurs if the host's cluster has active health checking configured"